src/components/application_manager/src/commands/mobile/show_constant_tbt_request.cc

```diff
@@ -183,27 +183,17 @@ void ShowConstantTBTRequest::on_event(const event_engine::Event& event) {
   switch (event.id()) {
     case hmi_apis::FunctionID::Navigation_ShowConstantTBT: {
       LOG4CXX_INFO(logger_, "Received Navigation_ShowConstantTBT event");
-      std::string return_info;
 
       mobile_apis::Result::eType result_code =
           GetMobileResultCode(static_cast<hmi_apis::Common_Result::eType>(
               message[strings::params][hmi_response::code].asInt()));
-      HMICapabilities& hmi_capabilities =
-          application_manager_.hmi_capabilities();
-      bool result = false;
-      if (mobile_apis::Result::SUCCESS == result_code) {
-        result = true;
-        return_info =
-            message[strings::msg_params][hmi_response::message].asString();
-      } else if ((mobile_apis::Result::UNSUPPORTED_RESOURCE == result_code) &&
-                 hmi_capabilities.is_ui_cooperating()) {
-        result = true;
-      }
-
-      SendResponse(result,
-                   result_code,
-                   return_info.empty() ? 0 : return_info.c_str(),
-                   &(message[strings::msg_params]));
+      bool result = mobile_apis::Result::SUCCESS == result_code;
+      result =
+          result || (mobile_apis::Result::UNSUPPORTED_RESOURCE == result_code &&
+                     (HmiInterfaces::STATE_AVAILABLE ==
+                      application_manager_.hmi_interfaces().GetInterfaceState(
+                          HmiInterfaces::HMI_INTERFACE_Navigation)));
+      SendResponse(result, result_code, NULL, &(message[strings::msg_params]));
```

Comment:
@dtrunov , in old code there was info param `return_info.empty() ? 0 : return_info.c_str(),` . Now there is `NULL`. Are you sure that it will not broke some test cases?